CHANGELOG.md

```diff
@@ -56,6 +56,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Renamed IoU -> Jaccard Index ([#662](https://github.com/PyTorchLightning/metrics/pull/662))
 
 
+- Renamed audio SDR metrics: ([#711](https://github.com/PyTorchLightning/metrics/pull/711))
+  * `functional.sdr` -> `functional.signal_distortion_ratio`
+  * `functional.si_sdr` -> `functional.scale_invariant_signal_distortion_ratio`
+  * `SI_SDR` -> `ScaleInvariantSDR`
```

Comment:
Also inconsistency between class and functional interface (similar to #712 ): either sdr or signal_distortion_ratio